tests/dummy/app/templates/demos/sl-menu.hbs

```diff
@@ -194,7 +188,7 @@ export default Ember.View.extend({
 
         // need to capture TAB key with keydown event
         Ember.$( document ).on( 'keydown.menu', function( e ) {
-            if ( 9 === e.keyCode && this.get( 'controller.keyboardInUse' ) ) {
+            if ( 9 === e.keyCode &amp;&amp; this.get( 'controller.keyboardInUse' ) ) {
                 e.preventDefault();
 
```

Comment:
These code examples should also be updated to reflect the new reality in the file they are excerpted from - on didInsert Element was moved, etc.

Also, the listed path locations for these files needs to be updated to reflect the "demo" folder they are now within